doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Ansible/Helm add api command proposal for Operator SDK
+authors:
+  - "@bharathi-tenneti"
+reviewers:
+  - "@fabianvf"
+  - "@cmacedo"
+approvers:
+  - "@fabianvf"
+  - "@jlanford"
+  - "@dmesser"
+creation-date: 2020-03-10
+last-updated: 2020-03-15
+status: implementable
+---
+
+
+# Ansible/Helm add API command proposal for Operator SDK
+
+
+## Release Signoff Checklist
+
+- \[x\] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Accceptance criteria
+- \[ \] User-facing documentation is created
+
+
+
+## Summary
+
+The proposal is to enable Ansible/Helm operator developers to create additional APIs, through SDK CLI.
+
+## Motivation
+
+As of today, SDK CLI can be used only to add additonal APIs for Go based Operators.
+Ansible/Helm operator developers are not able to create additonal APIs via CLI, once the original project scaffolds. Today, developers have to manually add necessary files to the project scaffold.
+
+## Goals
+
+* Ansible/Helm operator developer can use existing SDK CLI commands to create additonal APIs as needed.
+* Ansible/Helm operator developer should be able to use flags necessary for Ansible/Helm as used in `operator-sdk new`CLI. for adding additional APIs as well.
+* Ansible/Helm operator developer can find supported documentation for the same.
+
+## Non-Goals
+
+* Updating Molecule tests for additional APIs created for Anible based operators.[**TBD**]
+* Generating/Updating Playbook for additional APIs for Ansible based operators. [**TBD**]
+
```

Comment:
Are you ok with @fabianvf, should we not to do both?

Reply:
I'm fine with this, for developers these parts would be the bits they understand, handling the CRD stuff is the most useful